### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, mkdir 'C:\Users\c\AppData\Local\atom\app-1.1.0...

### ISSUE THREAD

**Issue by sichvoge** [mulesoft/api-workbench#30]
[Enter steps to reproduce below:]

1. create new raml with 'Create RAML Project' modal

**Atom Version**: 1.1.0
**System**: BBB
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package, v0.8.5

### Stack Trace

Uncaught Error: ENOENT: no such file or directory, mkdir 'C:\Users\c\AppData\Local\atom\app-1.1.0\undefined\newRamlProject'

```
At fs.js:799

Error: ENOENT: no such file or directory, mkdir 'C:\Users\c\AppData\Local\atom\app-1.1.0\undefined\newRamlProject'
  at Error (native)
  at Object.fs.mkdirSync (fs.js:799:18)
  at e.onOk (C:\Users\c\.atom\packages\api-workbench\main.js:28:20052)
  at C:\Users\c\.atom\packages\api-workbench\main.js:28:19789
  at C:\Users\c\.atom\packages\api-workbench\main.js:26:6990
  at Array.forEach (native)
  at HTMLButtonElement._onClickListeners.length._onAltClickListeners.length.e.onclick (C:\Users\c\.atom\packages\api-workbench\main.js:26:6963)

```

### Commands

```
     -0:16.7.0 api-workbench:newProject (ul.list-inline.tab-bar.inset-panel)
```

### Config

```json
{
  "core": {},
  "api-workbench": {}
}
```

### Installed Packages

```coffee
# User
api-workbench, v0.8.5
linter, v1.10.0

# Dev
No dev packages
```

**Comment by sichvoge** [comment]
Seems the workbench executes `mkdir` on Windows which does not work with child nodes.

**Comment by ddenisenko** [comment]
Fixed. The fix will be published in 0.8.6.
